packages/playground/blueprints/src/lib/steps/activate-theme.ts

```diff
@@ -0,0 +1,35 @@
+import { StepHandler } from '.';
+import { asDOM } from './common';
+
+export interface ActivateThemeStep {
+	step: 'activateTheme';
+	themeFolderName: string;
+}
+
+/**
+ * Activates a WordPress theme in the Playground.
+ *
+ * @param playground The playground client.
+ * @param themeFolderName The theme folder name.
+ */
+export const activateTheme: StepHandler<ActivateThemeStep> = async (
+	playground,
+	{ themeFolderName },
+	progress
+) => {
+	progress?.tracker.setCaption(`Activating ${themeFolderName}`);
+	const themesPage = asDOM(
+		await playground.request({
+			url: '/wp-admin/themes.php',
+		})
+	);
+
+	const link = themesPage.querySelector(
```

Comment:
Could we potentially call `switch_theme()` directly, instead of having to do this DOM manipulation stuff?

Reply:
Done! https://github.com/WordPress/wordpress-playground/pull/351/commits/24821ee5de866878362dcfee531eda8fd22a4113